routes.go

```diff
@@ -29,6 +29,7 @@ func (s *Server) Routes() {
 	s.Router.HandleFunc("/catholic-dioceses/per-decade/", s.CatholicDiocesesPerDecadeHandler()).Methods("GET", "HEAD")
 	s.Router.HandleFunc("/ne/northamerica/", s.NENorthAmericaHandler()).Methods("GET", "HEAD")
```

Comment:
Will you be removing these handlers once you can change the visualizations to use the new one?

Reply:
Yep, there's no reason to keep those around. I think the only site that I'll need to update for the new endpoint is Religious Ecologies (the diocese map and cities map).